src/components/IdCard.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+
+function IdCard(props) {
+
+    const { lastName, firstName, gender, height, birth, picture } = props;
+  
+    return (
+    <div className='info-card'>
+        <div>
+            <img src={picture}></img>
+        </div>
+        <div>
+            <p><strong>First name: </strong>{firstName}</p>
+            <p><strong>Last name: </strong>{lastName}</p>
+            <p><strong>Gender: </strong>{gender}</p>
+            <p><strong>Height: </strong>{height}</p>
+            <p><strong>Birth: </strong>{birth.toLocaleString()}</p>
```

Comment:
Here, to not display the hours, you could've used slice, like:

`  <p><strong>Birth: </strong>{birth.toLocaleString().slice(0,10)}</p>`